packages/bridge/src/vl/time-series/TimeSeries.ts

```diff
@@ -174,31 +190,26 @@ export class TimeSeries {
       return this._viz.variables[this._variableName];
     }
 
-    this._viz.variables[this._variableName] = this._propertyName &&
+    if (this._propertyName &&
       this._viz[this._propertyName] &&
-      this._viz[this._propertyName].isAnimated()
-        ? this._viz[this._propertyName]
-        : this._createDefaultAnimation();
+      this._viz[this._propertyName].isAnimated()) {
+        return this._viz[this._propertyName];
+    }
 
-    return this._viz.variables[this._variableName];
+    return this._createDefaultAnimation();
   }
 
   private _createDefaultAnimation() {
     const s = this._carto.expressions;
-    const animation = s.animation(
-      s.linear(
-        s.prop(this._columnName),
-        s.globalMin(s.prop(this._columnName)),
-        s.globalMax(s.prop(this._columnName))
-      ),
+
+    return s.animation(
+      s.linear(s.prop(this._columnName)),
```

Comment:
s.linear does the globalMin and globalMax by default, right?

Reply:
Exactly 👌